cmd/oci-runtime-tool/validate.go

```diff
@@ -225,8 +210,8 @@ func checkProcess(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string
 
 	for index := 0; index < len(process.Capabilities); index++ {
 		capability := process.Capabilities[index]
-		if !capValid(capability) {
-			msgs = append(msgs, fmt.Sprintf("capability %q is not valid, man capabilities(7)", process.Capabilities[index]))
+		if !capValid(capability, hostCheck) {
+			msgs = append(msgs, fmt.Sprintf("capability %q is not valid, man capabilities(7)", capability))
```

Comment:
Complaining by capability (and not by index) makes sense to me.  But now that we're doing that, I think we should replace the current for loop with:

    for _, capability := range process.Capabilities {